src/test/run-make/thumb-none-cortex-m/Makefile

```diff
@@ -0,0 +1,35 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# How to run this
+# $ ./x.py clean
+# $ ./x.py test --target thumbv6m-none-eabi,thumbv7m-none-eabi src/test/run-make
+
+# Supported targets:
```

Comment:
:confetti_ball:

Reply:
Yes, we now build required artifacts in `./x.py test` execution.